.github/workflows/build.yml

```diff
@@ -0,0 +1,23 @@
+name: Java CI
+
+on:
+  pull_request:
+    branches:
+      - '*'
+  push:
+    branches:
+      - '*'
+jobs:
+    build:
+      runs-on: ubuntu-latest
```

Comment:
Try to run CI on different environments.
For example
```
        strategy:
            matrix:
                os: [ubuntu-latest, windows-latest, macos-latest]
        runs-on: ${{ matrix.os }}
```

Reply:
Changed, it's all gone